source/_posts/generating-pdf-in-ruby-on-rails.md

```diff
@@ -0,0 +1,197 @@
+---
```

Comment:
keyword is not linked with the main website, it should be linked with main website where you want to redirect it

Reply:
I don't know what this means.

Reply:
Your 1-2 tag should be linked with Nayan website (hyperlink should be there) for redirection to main website